src/nile/utils/get_nonce.py

```diff
@@ -0,0 +1,25 @@
+"""Retrieve and manage deployed accounts."""
+import logging
+import os
+import subprocess
+
+from nile.common import get_gateway
+
+GATEWAYS = get_gateway()
+
+
+def get_nonce(contract_address, network):
+    """Get the current nonce for contract address in a given network."""
+    command = ["starknet", "get_nonce", "--contract_address", contract_address]
+
+    if network == "mainnet":
+        os.environ["STARKNET_NETWORK"] = "alpha-mainnet"
+    elif network == "goerli":
+        os.environ["STARKNET_NETWORK"] = "alpha-goerli"
+    else:
+        command.append(f"--feeder_gateway_url={GATEWAYS.get(network)}")
+
+    nonce = int(subprocess.check_output(command).strip())
+
+    logging.info(f"\nCurrent nonce for {contract_address} is {nonce}")
```

Comment:
Moving logging to CLI would break the consistency with the other commands logging from the implementation and not from the CLI. To fix the issue, I separated the log from the logic in the get-nonce.py file, and I'm using the logic without logging in account.send, let me know what you think.

Reply:
Ahh yes, I see the consistency issue. This solution looks good to me!